scc/carvel-package-supply-chain.hbs.md

```diff
@@ -84,12 +85,21 @@ spec:
       - kapp: {}
 ```
 
-The Carvel `Package` generated by the Supply Chain has three configurable parameters:
+The Carvel `Package` generated by the Supply Chain has different configurable parameters
+based on workload type:
 
+`Server` workload type:
 - `replicas`: Number of pods that you want for the `apps/v1 Deployment`. Default is `1`.
-- `hostname`: Host name for the `networking.k8s.io/v1 Ingress`. Default is `example.com`.
+- `hostname`: Host name for the `networking.k8s.io/v1 Ingress`. If hostname is blank, no ingress
```

Comment:
>hostname is blank

Does this mean empty string or completely committed from the values file?

Reply:
I think something like this makes it more clear:
"If you don't need ingress, leave `hostname` as an empty string. If `hostname` is not included in the file, you see an error."

Reply:
done